syntax = "proto3";

message UserInfo 
{
    string user_name = 1;
    string user_password = 2;
}

message ResponseInfo 
{
    string message = 1;
    int32 response_id = 2;
    int32 yourfd = 3;
}

message PlayerInput
{
    string playerName = 1;
    int32 yaw = 2;
    int32 hori = 3;
    int32 vect = 4;
    int32 speed = 5;
    int32 type = 6;
    int32 characterId = 7;
    int32 skillId = 8;
}

message FrameData
{
    int32 frameNo = 1;
    PlayerInput playerInput = 2;
}
